index.html

```diff
@@ -0,0 +1,390 @@
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+
+                  <!-- Webpage title -->
+
+    <title>Youtube Replica</title>
+  </head>
+  <body>
+
+                       <!-- The top fixed Navigation Bar -->
+
+    <header>
+      <nav>
+        <div class="logo">
+          <a href="#" class="nav-buttons" style="cursor: pointer;">
+            <i class="material-icons">reorder</i>
+          </a>
+          <a href="#" class="nav-buttons">
+            <img
+              alt=""
+              class="logo"
+              src="yt_logo_rgb_light.png"
+              width="70"
+              height="20"
+            />
+          </a>
+        </div>
+
+        <form>
+          <div class="search-form">
+            <input
+              id="search-input"
+              type="text"
+              placeholder="Search.."
+              name="search"
+            />
+            <button id="submit-button" type="submit">
+              <i class="fa fa-search"></i>
+            </button>
+          </div>
+        </form>
+
+        <div class="right-side-icons">
+          <a href="#" style="cursor: pointer;" class="nav-buttons">
+            <i class="material-icons">videocam_off</i>
+          </a>
+
+          <a href="#" style="cursor: pointer;" class="nav-buttons">
+            <i class="material-icons">apps</i>
+          </a>
+          <a href="#" style="cursor: pointer;" class="nav-buttons">
+            <i class="material-icons">notifications_active</i>
+          </a>
+          <a href="#" style="cursor: pointer;" class="nav-buttons">
+            <i class="material-icons">account_circle</i>
+          </a>
+        </div>
+      </nav>
+    </header>
+
+                            <!-- The Main Content -->
+
+    <main class="wrapper">
+      <section class="main">
+
+                            <!-- Youtube Video Player -->
+
+        <section class="main-video">
+          <iframe
+            src="https://www.youtube.com/embed/0Wf9bRXUe8M"
+            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+            allowfullscreen
+          >
+          </iframe>
+        </section>
+
+                      <!-- Additional Info about the video -->
+
+                        <!-- Video Title -->
+
+
+        <h3>CSS Grid layout Crash Course</h3>
+
+                             <!-- Views, Likes, Dislikes and Share -->
+
+        <div class="view-history">
+          <span class="view">1,279,452 Views. Aug 1,2017 </span>
+          <span class="like-history">
+            <i
+              class="material-icons"
+              style="cursor: pointer; margin-left: 10px; margin-right: 10px;"
+              >thumb_up</i
+            >
+            500
+            <i
+              class="material-icons"
+              style="cursor: pointer; margin-left: 10px; margin-right: 10px;"
```

Comment:
- [ ] Kindly remove all the inline selectors since they are not a good practice.